content/en/docs/kubernetes/collector/components.md

```diff
@@ -661,3 +663,41 @@ subjects:
     name: otel-collector-opentelemetry-collector
     namespace: default
 ```
+
+## Prometheus Receiver
+
+Prometheus is a common format for Kubernetes metrics. The Prometheus receiver is
+meant to be a minimal drop-in replacement for the collection of those metrics.
+It supports the full set of Prometheus scrape_config options.
```

Comment:
```suggestion
It supports the full set of Prometheus `scrape_config` options.
```

Or maybe link to https://prometheus.io/docs/prometheus/1.8/configuration/configuration/#scrape_config?

Reply:
Linked